packages/end-to-end-testing/lib/chains.ts

```diff
@@ -0,0 +1,243 @@
+import { getAddress, isNetworkName, ZetaNetworkName } from "@zetachain/addresses";
+import { isLocalNetworkName, NetworkName } from "@zetachain/addresses/src/addresses.helpers";
+import { getConnectorFactory, getTokenFactory } from "@zetachain/protocol-contracts/lib/contracts.helpers";
+import {
+  ZetaConnectorEth,
+  ZetaConnectorNonEth,
+  ZetaEth,
+  ZetaNonEth,
+} from "@zetachain/protocol-contracts/typechain-types";
+import { Axios, AxiosInstance } from "axios";
+import * as dotenv from "dotenv";
+import { Overrides, providers, Signer, Wallet } from "ethers";
+import { ethers } from "hardhat";
+const axios = require("axios").default;
+// dotenv.config();
+
+// console.debug = function () {}; // Disables Debug Level Logging
+// console.info = function () {}; // Disables Info Level Logging
+
+const defaultOverrideOptions: Overrides = {
+  gasLimit: 2500000,
+  gasPrice: 20000000000,
+};
+
+export class Blockchain {
+  name: NetworkName | ZetaNetworkName;
+  chainId: number;
+  rpcEndpoint: string;
+  initStatus: undefined | Promise<void>;
+  p: any;
+  accounts: any;
+  api: AxiosInstance;
+  type: ZetaNetworkName;
+  wallet: Wallet;
+  zetaNetworkName: ZetaNetworkName;
+
+  constructor(
+    name: NetworkName | ZetaNetworkName,
+    rpcEndpoint: string,
+    chainId: number,
+    zetaNetworkName: ZetaNetworkName
+  ) {
+    this.name = name;
+    this.chainId = chainId;
+    this.rpcEndpoint = rpcEndpoint;
+    this.zetaNetworkName = zetaNetworkName;
+  }
+}
+
+export class EVMChain extends Blockchain {
+  connectorContractAddress: string;
+  connectorContract: ZetaConnectorEth | ZetaConnectorNonEth;
+  zetaContractAddress: string;
+  zetaContract: ZetaEth | ZetaNonEth;
+  signer: Signer | any;
+  name: NetworkName;
```

Comment:
Can we order the properties alphabetically? There's this plugin:
<img width="341" alt="image" src="https://user-images.githubusercontent.com/7400784/175028056-1c6584c6-92f9-4446-893b-ef63b54e6986.png">

It makes it really easy, just `cmd + shift + a`